spec/actions/app_spec.cr

```diff
@@ -7,84 +7,140 @@ module DppmRestApi::Actions::App
       it "responds with 401 Unauthorized" do
         get fmt_route "/some-app/config/key"
         response.status_code.should eq 401
+        ErrorResponse.from_json(response.body)
```

Comment:
All this lines can be a a helper method, like `assert_error_response(response)`

Reply:
(inspired by https://github.com/crystal-lang/crystal/blob/master/spec/std/json/pull_parser_spec.cr#L88)

Reply:
Implemented.